style(index.tsx): App fixed spacing issue in calibration modals

aptrons22 opened the pull request:
<!--
Thanks for taking the time to open a pull request! Please make sure you've read the "Opening Pull Requests" section of our Contributing Guide:

https://github.com/Opentrons/opentrons/blob/edge/CONTRIBUTING.md#opening-pull-requests

To ensure your code is reviewed quickly and thoroughly, please fill out the sections below to the best of your ability!
-->

# Overview
Closes #10677 
<!--
Use this section to describe your pull-request at a high level. If the PR addresses any open issues, please tag the issues here.
-->

# Changelog

- Removed code from DirectionControl.tsx file and placed in index.tsx

# Review requests
Low
<!--
Describe any requests for your reviewers here.
-->

# Risk assessment
Low

aptrons22 committed:
added margin bottom spacing btw down arrow svg and bottom button

aptrons22 committed:
added 1.5 rem spacing btw down arrow and bottom button

aptrons22 committed:
revised marginBottom spacing for calibration modals

koji commented on app/src/molecules/JogControls/index.tsx:
```diff
@@ -13,6 +13,7 @@ import {
 
 import type { Jog, Plane, StepSize } from './types'
 import type { StyleProps } from '@opentrons/components'
+import { SPACING } from '@opentrons/components'
```
`SPACING` should be imported in line 4

aptrons22 committed:
style(index.tsx): revised the index.tsx to include SPACING btw arrow svg & button

? Provide a longer description of the change: (press enter to skip)

#10677

aptrons22 commented:
> If I recall correctly, this pr addresses some comments @b-cooper had about adding the `marginBottom` to the `DirectionControl`. Should the reversion of that change be included here? Also, do we want to include this styling in the `JogControl` component itself, or do we want to pass it to all instances of that component that needs the spacing?

aptrons22 committed:
removed spacing from DirectionControl.tsx file

koji commented on app/src/molecules/JogControls/DirectionControl.tsx:
```diff
@@ -167,3 +167,5 @@ export function DirectionControl(props: DirectionControlProps): JSX.Element {
     </ControlContainer>
   )
 }
+
```
For line 170 and 171
```suggestion

```

jerader reviewed (approved):
looks good! 🦖

koji reviewed (approved):
The changes look good to me.
Left a comment on a small thing.
before merging, please fix the lint error.

```zsh
npx prettier --write app/src/molecules/JogControls/DirectionControl.tsx
```

After that you can 
```zsh
make lint-js
```
The error should be gone.

aptrons22 committed:
Corrected linting
